cbor-schema/plt.cddl

```diff
@@ -81,22 +87,26 @@ token-governance-operation = token-mint
     / token-burn
     / token-update-list
 
+; A token governance event, which is the result of executing a token governance operation.
+token-governance-event = token-supply-update-details
+    / token-list-update-details
```

Comment:
I don't think we should define a union type for these, as the event `type` string will determine how the details should be parsed.

Reply:
yeah that makes sense, will remove this